sdk/areas/areas.go

```diff
@@ -96,12 +97,12 @@ func (c *Client) GetArea(ctx context.Context, userAuthToken, serviceAuthToken, c
 	if next, ok := body["next"]; ok && (serviceAuthToken != "" || userAuthToken != "") {
 		b, err = json.Marshal(next)
 		if err != nil {
-			return
+			return areaDetails, err
 		}
 	}
 
 	err = json.Unmarshal(b, &areaDetails)
-	return
+	return areaDetails, err
```

Comment:
The error should always be nil here so make it explicit.
```suggestion
	return areaDetails, nil
```

Reply:
Not sure I get this comment. This is the err from the previous line I think ```err = json.Unmarshal(b, &areaDetails)``` so the error is not always nil. To avoid confusion I can do this like ```return  json.Unmarshal(b, &areaDetails)```, what would you prefer ?

Reply:
Ah, you're right, my mistake. OK, so this will probably be fine. But I'd move the `var areaDetails AreaDetails` to just before the Unmarshal so it clearly shows it'd be empty if there would be an error.
```
        var areaDetails AreaDetails
	err = json.Unmarshal(b, &areaDetails)
	return areaDetails, err
```